linkerd/app/src/env.rs

```diff
@@ -1008,7 +1008,7 @@ fn parse_default_policy(
         "all-authenticated" => {
             Ok(inbound::policy::defaults::all_authenticated(detect_timeout).into())
         }
-        "all-unauthenticated" => {
+        "all-unauthenticated" | "audit" => {
```

Comment:
question: if we treat this as an `all-unauthenticated` by default, when we create the default policy will it still have the correct metadata?

What would happen if for example we have an `audit` default policy and we get an inbound connection? Will we still log with the audit labels?